functions/Find-DbaUserObject.ps1

```diff
@@ -37,292 +37,294 @@ You should have received a copy of the GNU General Public License
 along with this program.  If not, see <http://www.gnu.org/licenses/>.	
 
 .LINK
-https://dbatools.io/Get-DbaStoredProcedure
+https://dbatools.io/Find-DbaUserObject
 
 .EXAMPLE
-Get-DbaStoredProcedure -SqlServer DEV01 -Value "html" -Verbose
-
-Checks in all user databases stored procedures for "html" in the textbody
+Find-DbaUserObject -SqlInstance DEV01 -Pattern ad\stephen
 
+Searches user objects for owner ad\stephen
+	
 .EXAMPLE
-Get-DbaStoredProcedure -SqlServer DEV01 -Database MyDB -Value "html" -Verbose
+Find-DbaUserObject -SqlInstance DEV01 -Verbose
 
-Checks in "mydb" database stored procedures for "html" in the textbody
+Shows all user owned (non-sa, non-dbo) objects and verbose output 
 #>
 	[CmdletBinding()]
 	Param (
 		[parameter(Position = 0, Mandatory = $true, ValueFromPipeline = $True)]
-		[Alias("ServerInstance", "SqlInstance", "SqlServers")]
-		[string[]]$SqlServer,
+		[Alias("ServerInstance", "SqlServer", "SqlInstances")]
+		[object[]]$SqlInstance,
 		[System.Management.Automation.PSCredential]$SqlCredential,
```

Comment:
Thanks for the review!!

You'll find that some commands do have Credential. We have decided in dbatools.io/tstyle that we'll be reserving Credential for Windows/Linux and SqlCredential for SQL Server itself.